lib/Expression_gen.ml

```diff
@@ -19,12 +19,11 @@ let stan_namespace_qualify f =
 
 (* return true if the types of the two expression are the same *)
 let types_match e1 e2 = e1.emeta.mtype = e2.emeta.mtype
+let is_stan_math f = ends_with "__" f || starts_with "stan::math::" f
 
 (* "__" is an illegal suffix for user functions, used for built-in operators not in signatures *)
 let is_user_defined f =
```

Comment:
Isn't this just checking whether it's Mir.UserDefined or not?

Reply:
Yeah, this is now only called via `gen_extra_fun_args` which itself is only called for `UserDefined`. I'll remove.